### ISSUE TITLE
Fatal Error On v2

### ISSUE THREAD

**Issue by theskyfloor** [craftcms/webhooks#8]
When I try to create a new hook or view an existing on v2 I get a fatal exception:

```
PHP Fatal Error – yii\base\ErrorException

Class Solspace\Freeform\Library\Integrations\PaymentGateways\PaymentGatewayOAuthConnector   contains 11 abstract methods and must therefore be declared abstract or implement the remaining methods   (Solspace\Freeform\Library\Integrations\PaymentGateways\AbstractPaymentGatewayIntegration::fetchPlans,  Solspace\Freeform\Library\Integrations\PaymentGateways\AbstractPaymentGatewayIntegration::createPlan, Solspace\Freeform\Library\Integrations\PaymentGateways\AbstractPaymentGatewayIntegration::fetchPlan, ...)
```

Seems related to Freeform. Not sure if I should report to them or here : )

**Comment by brandonkelly** [comment]
Thanks for letting me know! I’ve reported the invalid class to Solspace, but will go ahead and work on a Webhooks update that offloads component class discovery to an Ajax request, so at least this sort of bug won’t stop you from using the plugin.

**Comment by brandonkelly** [comment]
Just released 2.0.1 with that change.
